content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,107 @@
 Italy
 =====
 
-Allow the user to generate the EDI document for Italian invoicing.
+Configuration
+=============
+The first necessary step to carry out accounting transactions in Italy is the
+:guilabel:`Italy - Accounting`. Once this module is installed you can start managing your accounting
+in Odoo. For all B2B transactions, Italy requires electronic invoicing, the use of specific formats and the
+transmission of invoices on **SDI** platform.
+
+Odoo modules allow to:
 
-This module allows the creation of the EDI documents and the communication with the SDICoop web
-service of FatturaPA to send or receive invoices and notifications.
+- Generate electronic invoices in **FatturaPA's** format.
+- Send and receive electronic invoicing through SDIcoop.
+- Use the **Codice Destinatario** `K95IV18`.
```

Comment:
Well, this third is just something that you have to do, not something that "Odoo allows you to".
I would just remove the third bullet point and use the other two.